gsb_feature_program_ct.strongarm.inc

```diff
@@ -324,10 +324,6 @@ function gsb_feature_program_ct_strongarm() {
             'weight' => '21',
             'visible' => FALSE,
           ),
-          'program_explore_other' => array(
```

Comment:
This program_explore_other 'shouldn't be in here, it is that the gsb_feature_program_ct had a dependency on that field at one time. The program_explore_other field was removed, however the feature hasn't been rebuilt in the meantime.

![verified](https://user-images.githubusercontent.com/654993/36928089-675006ee-1e37-11e8-8f9d-65b718e7ced7.png)

In my work in WP-8109 I came across these same changes in this file. Hopefully soon once the features are synced up, you won't have so much extraneous information.

Reply:
Should I scrap this branch then and redo it or do I manually remove the program_explore_other?

Reply:
The code I put in for WP-8109 went through each of the 'overridden' features and created a new version that included the overriding parts. Because your feature export included the code for both your intended updates and the overrides (the program_explore_other) bits, I removed gsb_feature_program_ct from the updates I'm pushing on WP-8109. So you can just push this as it is